py-scripts/lf_report.py

```diff
@@ -532,26 +528,25 @@ def copy_allure_report(self):
         try:
             files = os.listdir(self.allure_report_history)
             for fname in files:
-                allure_report_history_file =  str(self.allure_report_history_path) + '/' + str(fname)
+                allure_report_history_file = str(self.allure_report_history_path) + '/' + str(fname)
                 allure_results_history_file = str(self.allure_results_history_path) + '/' + str(fname)
-                shutil.copy(allure_report_history_file,allure_results_history_file)
+                shutil.copy(allure_report_history_file, allure_results_history_file)
 
         except Exception as x:
             traceback.print_exception(Exception, x, x.__traceback__, chain=True)
             logger.info("Either no allure report present or the copy of history failed.")
 
-
-
     def generate_allure_report(self):
         # TODO current the junit.xml is placed in the base directory
         # allure report directory needs to be allure-report
-        allure_command = "allure generate {allure_results} --report-dir {allure_report} --clean".format(allure_results=self.allure_results,allure_report=self.allure_report_dir)
-        summary_output = ''
+        allure_command = "allure generate {allure_results} --report-dir {allure_report} --clean".format(allure_results=self.allure_results, allure_report=self.allure_report_dir)
+        summary_output = ''                                                                                 # noqa F841 - unused variable
```

Comment:
Does this pass w/ all the whitespace after the conditional? If so, let's just get it to two spaces after, as flake checks for minimum of